fixtures/uitests/tests/ui/interface_not_sync_and_send.stderr

```diff
@@ -0,0 +1,32 @@
+error[E0277]: `Cell<u32>` cannot be shared between threads safely
+   --> $DIR/counter.uniffi.rs:119:1
+    |
+119 | uniffi::deps::static_assertions::assert_impl_all!(Counter: Sync, Send);
+    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    | |
+    | `Cell<u32>` cannot be shared between threads safely
+    | required by this bound in `assert_impl_all`
+    |
+    = help: within `Counter`, the trait `Sync` is not implemented for `Cell<u32>`
+    = note: required because it appears within the type `Counter`
+    = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
```

Comment:
This is the error produced by the new assertion, which I think does a good job of helping the user find the underlying problem.